### ISSUE TITLE
show-resource does not work for files

### ISSUE THREAD

**Issue by bertsky** [OCR-D/core#752]
We have a catch in the implementation of `show_resource` for files: We always write to `sys.stdout.buffer`, which works for ttys  – where it is almost always useless, because no-one wants to look at binaries –, but with a file redirect it ceases to work. 

I have not been able to get it working with `flush()` or `close()`, or doing `sys.stdout.detach()` instead of using its `.buffer` yet. Also, just doing everything in memory à la `os.fdopen(sys.stdout.fileno(), 'wb').write(f.read())` does not work either.

**Comment by bertsky** [comment]
Note: this is a real show-stopper for **preset** resources, because in that case we not only may want to _see_ the parameters as a human, but parse them (e.g. in a bashlib processor with presets).

@kba please add the bug label and prioritize over the other resource-related issues!

**Comment by kba** [comment]
I cannot reproduce this, for example if I do

```
ocrd-tesserocr-recognize -C deu.traineddata > foo
```

it does what I expect, read from `$XDG_DATA_HOME/ocrd-resources/ocrd-tesserocr-recognize/deu.traineddata` and write to `foo`. I've tested it in zsh, bash and dash, both interactive and in a script. What exactly goes wrong with file redirection for you? Is nothing written or is there an error?
```

**Comment by bertsky** [comment]
Cannot reproduce myself anymore. :confused:
